internal/providers/oraclecloud/oraclecloud.go

```diff
@@ -53,7 +53,8 @@ func fetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
 		RetryCodes: []int{http.StatusTooManyRequests},
 	})
 	if err != nil && err != resource.ErrNotFound {
-		return types.Config{}, report.Report{}, fmt.Errorf("fetching to buffer: %w", err)
+		// Do not wrap these errors, ignition uses direct comparsion to distinguish them.
```

Comment:
```suggestion
		// Do not wrap these errors, ignition uses direct comparison to distinguish them.
```

Though ideally we also mention `ErrNeedNet` here.

Presumably... we should consistently have that comment in the other providers as well for consistency. But obviously not new.